data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1932,9 +1932,18 @@ spec:
             - Passthrough
             - Manual
             type: string
+          customFeatures:
```

Comment:
Any other possible names? Would `features` or `featureGates` make sense? `

Reply:
I'm open to naming suggestions, `featureGates` would match the Kube types, so we can do that. Within OpenShift the API looks like
```
featureSet: CustomNoUpgrade
customNoUpgrade:
  enabled: []
  disabled: []
```
So perhaps it would be better to have something that suggests it's custom only? Maybe not??

Reply:
> So perhaps it would be better to have something that suggests it's custom only

+1. i am good with custom features. Brainstorming: what about `customFeatureSet`?